### ISSUE TITLE
CRAN check fails under macos, issue due to vignette builds fail for col_counts.Rmd

### ISSUE THREAD

**Issue by shajoezhu** [insightsengineering/rtables#886]
```
checking tests ... [78s/90s] OK
  Running ‘testthat.R’ [78s/89s]
checking for unstated dependencies in vignettes ... OK
checking package vignettes in ‘inst/doc’ ... OK
checking running R code from vignettes ... [50s/58s] ERROR
Errors in running code in vignettes:
when running code in ‘col_counts.Rmd’
  ...

> tbl4

  When sourcing ‘col_counts.R’:
Error: error in evaluating the argument 'x' in selecting a method for function 'toString': Detected different colcount visibility among sibling facets (those arising from the same split_cols_by* layout instruction). This is not supported.
Set count values to NA if you want a blank space to appear as the displayed count for particular facets.
First disagreement occured at paths:
ARM[A: Drug X]->SEX[F]
ARM[A: Drug X]->SEX[M]
Execution halted

  ‘advanced_usage.Rmd’ using ‘UTF-8’... [2s/2s] OK
  ‘baseline.Rmd’ using ‘UTF-8’... [1s/1s] OK
  ‘clinical_trials.Rmd’ using ‘UTF-8’... [8s/10s] OK
  ‘col_counts.Rmd’ using ‘UTF-8’... failed
  ‘custom_appearance.Rmd’ using ‘UTF-8’... [5s/6s] OK
  ‘example_analysis_coxreg.Rmd’ using ‘UTF-8’... [2s/3s] OK
  ‘exploratory_analysis.Rmd’ using ‘UTF-8’... [4s/4s] OK
  ‘format_precedence.Rmd’ using ‘UTF-8’... [2s/3s] OK
  ‘introduction.Rmd’ using ‘UTF-8’... [3s/3s] OK
  ‘introspecting_tables.Rmd’ using ‘UTF-8’... [1s/2s] OK
  ‘manual_table_construction.Rmd’ using ‘UTF-8’... [1s/1s] OK
  ‘sorting_pruning.Rmd’ using ‘UTF-8’... [6s/7s] OK
  ‘split_functions.Rmd’ using ‘UTF-8’... [2s/3s] OK
  ‘subsetting_tables.Rmd’ using ‘UTF-8’... [2s/3s] OK
  ‘tabulation_concepts.Rmd’ using ‘UTF-8’... [3s/4s] OK
  ‘tabulation_dplyr.Rmd’ using ‘UTF-8’... [2s/2s] OK
  ‘title_footer.Rmd’ using ‘UTF-8’... [4s/4s] OK
```

_Originally posted by @shajoezhu in https://github.com/insightsengineering/rtables/issues/885#issuecomment-2185418344_

**Comment by gmbecker** [comment]
head of main passes for me on a mac M1, will try exact version on cran

**Comment by shajoezhu** [comment]
closed by #896
